### ISSUE TITLE
Can't resolve '@metaplex-foundation/js' [Next.js 13]

### ISSUE THREAD

**Issue by imaai** [metaplex-foundation/js#422]
```zsh
error - ./app/page.tsx:4:0
Module not found: Can't resolve '@metaplex-foundation/js'
  2 | import styles from './page.module.css'
  3 | import solana from '@solana/web3.js'
> 4 | import {} from '@metaplex-foundation/js';
  5 | 
  6 | export default function Home() {
  7 |   return (
```

```json
# tsconfig.json
{
  "compilerOptions": {
    "target": "es5",
    "lib": ["dom", "dom.iterable", "esnext"],
    "allowJs": true,
    "skipLibCheck": true,
    "strict": true,
    "forceConsistentCasingInFileNames": true,
    "noEmit": true,
    "esModuleInterop": true,
    "module": "esnext",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "jsx": "preserve",
    "incremental": true,
    "plugins": [
      {
        "name": "next"
      }
    ]
  },
  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
  "exclude": ["node_modules"]
}
```

How to reproduce
- `yarn create next-app --experimental-app`
- `yarn add @metaplex-foundation/js @solana/web3.js`
- `import {} from '@metaplex-foundation/js'`
- `yarn dev`

**Comment by samuel-uj** [comment]
Same problem here. Force an older version.

**Comment by imaai** [comment]
@samuel-uj Older version of?

**Comment by samuel-uj** [comment]
Of the @metaplex-foundation/js module. `yarn add @metaplex-foundation/js@0.17.5` works for me.

**Comment by josip-volarevic** [comment]
> Of the @metaplex-foundation/js module. `yarn add @metaplex-foundation/js@0.17.5` works for me.

`v0.17.6` is also fine for me. I think `v0.17.7` release from 15 hours ago is corrupt in some way

**Comment by lorisleiva** [comment]
Hi there 👋 

Yes this was a build error in version `0.17.7` which has now been fixed in version `0.17.8`.

Apologies about that. 🙏
